### PR TITLE
Add option to change the size of internal ring buffers

### PR DESCRIPTION
**Pull request by gciria**
Signed-off-by: Gustavo Ciria <gdciria@gmail.com>

# What this PR does / why we need it

This PR enables changing the size of internal ring buffers:
- ring_size
- inputbuffer_ring_size

Depending on your infrastructure, these values need to be changed for better performance.

# Which issue this PR fixes

N/A
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

# Special notes for your reviewer

# Checklist

- [x] [DCO](https://github.com/KongZ/charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by KongZ** [comment]
Please fixed
```
Error: [trailing-spaces] trailing spaces
Error: Error linting charts: Error processing charts
```
https://github.com/KongZ/charts/runs/6502992722?check_suite_focus=true#step:9:35

**Comment by gciria** [comment]
Hi @KongZ , fixed spaces 😃 
note: I sent a wrong commit, already reverted.

File: charts/graylog/templates/configmap.yaml [hunk 1]
```diff
@@ -108,8 +108,16 @@ data:
     processbuffer_processors = 5
     outputbuffer_processors = 3
     processor_wait_strategy = blocking
+  {{- if .Values.graylog.options.ringSize }}
+    ring_size = {{ .Values.graylog.options.ringSize }}
```
Comment (KongZ) [inline charts/graylog/templates/configmap.yaml:112]: you can simplify by using default value
```
ring_size = {{ .Values.graylog.options.ringSize | default 65536 }}
```

File: charts/graylog/templates/configmap.yaml [hunk 2]
```diff
@@ -108,8 +108,16 @@ data:
...
+  {{ else }}
     ring_size = 65536
+  {{- end }}
+  {{- if .Values.graylog.options.inputBufferRingSize }}
+    inputbuffer_ring_size = {{ .Values.graylog.options.inputBufferRingSize }}
```
Comment (KongZ) [inline charts/graylog/templates/configmap.yaml:117]: here too
```
inputbuffer_ring_size = {{ .Values.graylog.options.inputBufferRingSize | default 65536 }}
```

Comment (gciria) [reply charts/graylog/templates/configmap.yaml]: you're right, thanks!
change sent

**Review by KongZ — APPROVED** [review]

**Pull request merged**